src/smokeTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/IdamConfigStatusTest.java

```diff
@@ -32,19 +31,17 @@ public void each_jurisdiction_should_have_valid_credentials() throws IOException
         ObjectMapper mapper = new ObjectMapper();
 
         mapper.readTree(response).elements().forEachRemaining(responseStatus -> {
-            JurisdictionConfigurationStatus status = new JurisdictionConfigurationStatus(
-                responseStatus.get("jurisdiction").asText(),
-                responseStatus.get("is_correct").asBoolean(),
-                responseStatus.get("error_description").asText()
-            );
-
-            assertThat(status.isCorrect)
-                .withFailMessage(
-                    "Misconfigured %s jurisdiction, error description: %s. Check the logs for more details",
-                    status.jurisdiction,
-                    status.errorDescription
-                )
-                .isTrue();
+                String jurisdiction = responseStatus.get("jurisdiction").asText();
+                boolean isCorrect = responseStatus.get("is_correct").asBoolean();
+                String errorDescription = responseStatus.get("error_description").asText();
+
+                assertThat(isCorrect)
+                    .withFailMessage(
+                        "Misconfigured %s jurisdiction, error description: %s. Check the logs for more details",
+                        jurisdiction,
+                        errorDescription
+                    )
+                    .isTrue();
```

Comment:
indentation of the body of this lambda changed, was this intentional?

Reply:
no, it must have been the IDE